src/components/Validations/ValidationSummary.tsx

```diff
@@ -1,15 +1,16 @@
 import * as React from 'react';
-import { ObjectValidation, ValidationTypes } from '../../types/IstioObjects';
+import { ValidationTypes } from '../../types/IstioObjects';
 import { style } from 'typestyle';
 import { Text, TextVariants, Tooltip, TooltipPosition } from '@patternfly/react-core';
 import Validation, { severityToValidation } from './Validation';
-import { higherSeverity, highestSeverity } from '../../types/ServiceInfo';
+import { CSSProperties } from 'react';
 
 interface Props {
   id: string;
-  validations: ObjectValidation[];
-  definition?: boolean;
-  size?: string;
+  errors: number;
+  warnings: number;
+  style?: CSSProperties;
```

Comment:
Side comment.
Something is not right with this abstraction if we need to pass to a component css style that requires to understand the implementation details of the component.

No action comment, but leaving a comment here.

Reply:
I'm not sure it needs to understand the impl, this just provides an override to the default style, which is a fairly common component feature, I think.  Maybe I misunderstand the comment.

Reply:
That is exactly it. I think is a common approach for styling components.

Reply:
No, it is not. As the "wrapper" prop affects as well in the final style.
But, as commented this is side comment about abstraction and best practices.
It doesn't require a change for this PR, but to me is a sympton that probably there are technical debt between layers.

Reply:
I guess I don't really understand.  How is what is done here different than other components that support a style property?  Maybe you can answer this is dev-list if you think it will benefit people.